src/js/trove/require-node-compile-dependencies.js

```diff
@@ -11,8 +11,8 @@ define("q", [], function() {return q;});
 jsmd5 = require("js-md5");
 define("js-md5", [], function() {return jsmd5;});
 
-canvas = require("canvas");
-define("canvas", [], function() {return canvas;});
+crossFetch = require("cross-fetch");
+define("cross-fetch", [], function() {return crossFetch;});
```

Comment:
Oh this is confusing as a diff.

We don't need canvas in the _compiler_, just in the runtime libraries. So it's an accident that it was included here. But maybe confusing for this PR.